doc/classes/Shortcut.xml

```diff
@@ -16,24 +16,24 @@
 				Returns the shortcut's [InputEvent] as a [String].
 			</description>
 		</method>
-		<method name="is_shortcut" qualifiers="const">
+		<method name="has_event" qualifiers="const">
 			<return type="bool" />
-			<argument index="0" name="event" type="InputEvent" />
 			<description>
-				Returns [code]true[/code] if the shortcut's [InputEvent] equals [code]event[/code].
+				If [code]true[/code], this shortcut has a valid [member event] assigned to it.
```

Comment:
Isn't "if true" something we usually use for arguments? The description could be something like "Returns whether the shortcut has a valid event".

Reply:
Fixed.